pkg/objectupdate/rte/rte.go

```diff
@@ -129,6 +129,13 @@ func DaemonSetHashAnnotation(ds *appsv1.DaemonSet, cmHash string) {
 	template.Annotations[hash.ConfigMapAnnotation] = cmHash
 }
 
+func DaemonSetAnnotation(ds *appsv1.DaemonSet, annotKey string, annotValue string) {
```

Comment:
this can be an overkill, but let's roll with it for now

Reply:
we can add it under the spec annotations; or inject a controller event (shown in the operator CR events) which policy is being used. If it doesn't make a difference why do you think this is an overkill?

Reply:
adding a new function in `pkg/objectupdate/rte` seems overkilly to me, but not enough to change course just yet

Reply:
We tend to use annotation to reflect some useful information which is not exposed officially by the API.

In this case the SELinux context policy exposed explicitly in the daemonset/pod spec so it's part of the official API, hence annotation is not needed.

Reply:
I might have missed this is already reflected. Can you point where this is set in the DS/pod spec please?